libraries/chain/database.cpp

```diff
@@ -2867,7 +2866,6 @@ namespace golos {
             add_core_index<feed_history_index>(*this);
             add_core_index<convert_request_index>(*this);
             add_core_index<liquidity_reward_balance_index>(*this);
-            add_core_index<operation_index>(*this);
             add_core_index<category_index>(*this);
```

Comment:
Remove `category_index` - it doesn't exist any more.

Reply:
Done